src/index.js

```diff
@@ -0,0 +1,164 @@
+  decButton: null,
+  skyDropdown: null,
+  resetButton: null,
+  realtimeButton: null,
+};
+
+const makeRealtimeTempUpdates = () => {
+  // calls proxy API to get lat and lon
+  getLatLon(state.cityName)
+    .then(({ latitude, longitude }) => {
+      // calls proxy API to get temp
+      getWeather(latitude, longitude)
+        .then(({ tempK }) => {
+          // update temp state
+          updateTempState(tempK);
+          updateTempOnWebpage();
+          changeColorAndLandscape();
+        })
+        .catch((error) => {
+          console.log('error in getWeather!');
+        });
+    })
+    .catch((error) => {
+      console.log('error in getLatLon!');
+    });
+};
+
+const getLatLon = async (city) => {
+  const latLonResponse = await axios.get('http://127.0.0.1:5000/location', {
+    params: {
+      q: city,
+    },
+  });
+  const latitude = latLonResponse.data[0].lat;
+  const longitude = latLonResponse.data[0].lon;
+  return { latitude, longitude };
+};
+
+const getWeather = async (apiLatitude, apiLongitude) => {
+  const weatherResponse = await axios.get('http://127.0.0.1:5000/weather', {
+    params: {
+      lat: apiLatitude,
+      lon: apiLongitude,
+    },
+  });
+  const tempK = weatherResponse.data.main.temp;
+  return { tempK };
+};
+
+const updateTempState = (tempInKelvin) => {
+  // converts temp to Farenheit and saves it to the state
+  state.temperature = Math.floor(((tempInKelvin - 273.15) * 9) / 5 + 32);
+};
+
+const increaseTemp = () => {
+  state.temperature += 1;
+  updateTempOnWebpage();
+  changeColorAndLandscape();
+};
+
+const decreaseTemp = () => {
+  state.temperature -= 1;
+  updateTempOnWebpage();
+  changeColorAndLandscape();
+};
+
+const updateTempOnWebpage = () => {
+  const websiteTemp = document.getElementById('temp');
+  websiteTemp.textContent = `${state.temperature}°F`;
+};
+
+const changeColorAndLandscape = () => {
+  const websiteTemp = document.getElementById('temp');
+  const websiteLandscape = document.getElementById('landscape');
+  if (state.temperature >= 80) {
+    websiteTemp.className = 'temp_red';
+    websiteLandscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (70 <= state.temperature && state.temperature < 80) {
+    websiteTemp.className = 'temp_orange';
+    websiteLandscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (60 <= state.temperature && state.temperature < 70) {
+    websiteTemp.className = 'temp_yellow';
+    websiteLandscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (50 <= state.temperature && state.temperature < 60) {
+    websiteTemp.className = 'temp_green';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (40 <= state.temperature && state.temperature < 50) {
+    websiteTemp.className = 'temp_teal';
+    websiteLandscape.textContent = '🌲☃️⛄️🌲⛄️❄️🌲❄️🌲☃️⛄️❄️🌲';
+  } else if (state.temperature < 40) {
+    websiteTemp.className = 'temp_aqua';
+    websiteLandscape.textContent = '🧊🧊⛄️🌬️⛄️❄️☃️❄️🌬️🧊⛄️❄️☃️';
+  }
+};
+
+const changeSkyType = () => {
```

Comment:
Same as above, how could we dry up this code and make this data-driven and get rid of if/else statements?